### DISCUSSION TITLE
SBC for Coral Dual Edge TPU

### DISCUSSION THREAD

**Discussion by aranw** [geerlingguy/sbc-reviews#44] [Q&A]
Hey 👋🏻  

I was wondering whether anyone has tested a SBC with the [M.2 Accelerator with Dual Edge TPU](https://coral.ai/products/m2-accelerator-dual-edgetpu)? 

Looking for a SBC that will support the dual tpu so I can run some projects with the TPU

**Comment by geerlingguy** [comment]
Please see: https://github.com/geerlingguy/raspberry-pi-pcie-devices/issues/318 — over in that repo, we are working on testing it on Pi 5. It didn't work on Pi 4 / CM4 though.

**Reply by aranw** [reply]
Ah nice yeah I'll subscribe to that. I've read on certain devices only one TPU becomes available. I'm guessing perhaps due to the way it's wired up but I'm not sure at that level it's not something I fully understand at the moment

**Reply by geerlingguy** [reply]
Yeah, the problem is this particular hardware uses a rarely-implemented feature, where the M.2 slot has two PCIe lanes wired up instead of the normal 1, and so the hardware connections are just missing on most boards, unless the vendor specifically implements the two lane support.

**Reply by aranw** [reply]
@geerlingguy ahh okay that makes sense thanks for explaining that
